index.bs

```diff
@@ -350,7 +350,7 @@ parallel</a>:
 2. Run the <a>request Picture-in-Picture algorithm</a> with |video|.
 3. If the previous step threw an exception, reject |promise| with that
     exception and abort these steps.
```

Comment:
```suggestion
3. If the previous step [=exception/throws|threw=] an exception, [=queue a task=] using |video|'s [=media element event task source=] to [=/reject=] |promise| with that exception and abort these steps.
```

Reply:
For both resolve and reject paths, WebKit and Chrome don't queue task. See https://github.com/WebKit/WebKit/blob/d36ed108e92fb350723da8e7a88e7a7e8be1ef45/Source/WebCore/Modules/pictureinpicture/HTMLVideoElementPictureInPicture.cpp#L81 and https://source.chromium.org/chromium/chromium/src/+/main:third_party/blink/renderer/modules/document_picture_in_picture/picture_in_picture_controller_impl.cc;l=124;drc=20135c10f0869fdefb75d990ec84143a649d84c3

Reply:
This should be:
```
The {{requestPictureInPicture()}} method, when invoked, MUST run the following steps:

1. Run the <a>request Picture-in-Picture algorithm</a> with [=this=].
2. If the previous step [=exception/throws|threw=] an exception, return [=a promise rejected with=] that exception.
3. return [=a promise resolved with=] [=this=]'s associated [=Picture-in-Picture window=].
```

As it never seems to go into parallel.